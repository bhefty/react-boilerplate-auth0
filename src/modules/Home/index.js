import React from 'react';

const Home = (props) => {
  return (
    <div>
      <h1>Home</h1>
      <p>This is a private page. You must be logged in to view it.</p>
    </div>
  );
}

export default Home;
