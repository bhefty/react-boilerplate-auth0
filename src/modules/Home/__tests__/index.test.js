import React from 'react';
import renderer from 'react-test-renderer'

import Home from '../index';

describe('Home', () => {
  test('Home renders without crashing', () => {
    const tree = renderer.create(
      <Home />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
