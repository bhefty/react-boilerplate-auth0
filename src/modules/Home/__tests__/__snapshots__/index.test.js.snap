exports[`Home Home renders without crashing 1`] = `
<div>
  <h1>
    Home
  </h1>
  <p>
    This is a private page. You must be logged in to view it.
  </p>
</div>
`;
